src/analytics/utils.ts

```diff
@@ -29,9 +28,6 @@ export async function getDeviceId(): Promise<string> {
     // set ID
     ls.device.set(['id'], deviceId);
 
-    // update Sentry user immediately
-    Sentry.setUser({ id: deviceId });
```

Comment:
Decided to move this out so that we can identify users in Segment and Sentry in the same place.